pkgs/by-name/my/mybbscan/package.nix

```
@@ -0,0 +1,35 @@
+{ lib, python3, stdenv, fetchFromGitHub, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "my-b-bscan";
+  version = "3.2.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "0xB9";
+    repo = "MyBBscan";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```